### PR TITLE
[Feature] User Management

### PR DESCRIPTION
**Pull request by mbiskho**

### CODE REVIEW

**Comment by mbiskho** [comment]
Feature included : 

- PBI-26 : add user
- PBI-27 : update user
- PBI-28 : view user list
- PBI-29 : delete user

**Review by MassiveMassimo — COMMENTED** [review]
Approved

File: .eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
```
Comment (MassiveMassimo) [inline .eslintrc.json:3]: Approved

**Review by MassiveMassimo — APPROVED** [review]
approved

**Pull request merged**